images/sdn/Dockerfile.rhel

```diff
@@ -10,6 +10,9 @@ COPY --from=builder /go/src/github.com/openshift/sdn/openshift-sdn-controller /u
 COPY --from=builder /go/src/github.com/openshift/sdn/openshift-sdn-cni /opt/cni/bin/openshift-sdn
 COPY --from=builder /go/src/github.com/openshift/sdn/host-local /usr/bin/cni/osdn-host-local
 
+FROM registry.ci.openshift.org/ocp/4.13:cli AS cli
```

Comment:
@mccv1r0 I think you want to move this up immediately above the previous FROM... not in the middle.